src/AutoRest.CSharp/Mgmt/Decorator/OperationExtensions.cs

```diff
@@ -152,7 +152,7 @@ private static int RankRequestPath(RequestPath requestPath)
 
         public static string GetHttpPath(this Operation operation)
         {
-            return operation.GetHttpRequest()?.Path ??
+            return operation.GetHttpRequest()?.Path.TrimEnd('/') ??
```

Comment:
```suggestion
            return operation.GetHttpRequest()?.Path?.TrimEnd('/') ??
```

Reply:
Path is defined as `public string Path { get; set; }`, so if `GetHttpRequest()` is not null, `Path` won't/can't be null. The null-conditional operators are [short-circuiting](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/member-access-operators#null-conditional-operators--and-). If `GetHttpRequest()` is null, the expression stops executing, so I guess we don't need the extra `?` here.